http-client/Network/HTTP/Client/Core.hs

```diff
@@ -270,3 +271,11 @@ httpRedirect' count0 http' req0 = go count0 req0 []
 -- Since 0.1.0
 responseClose :: Response a -> IO ()
 responseClose = runResponseClose . responseClose'
+
+-- | Perform an action using a @Connection@ acquired from the given @Manager@,
+--   considering the proxy server.
```

Comment:
A few points:

1. Is there anything proxy-specific about this code?
2. Can you elaborate in the Haddocks here what the use case is for this, and in particular than normal usage of the library for HTTP requests wouldn't need this function?
3. Please add a minor version bump to the cabal file, add a `@since` comment here, and update the ChangeLog.

Reply:
I've dropped words related to proxy from the doc and name. How about this?